src/services/db/GitFileSystemService.ts

```diff
@@ -1724,6 +1822,33 @@ export default class GitFileSystemService {
     })
   }
 
+  /**
+   * Does not create a new branch if it already exists
+   */
+  createLocalTrackingBranchIfNotExists(repoName: string, branchName: string) {
```

Comment:
can we add return types for the various functions? can see the other funcs for example

e.g. `: ResultAsync<string, GitFileSystemError | GitFileSystemNeedsRollbackError>`

Reply:
[dc591e7](https://github.com/isomerpages/isomercms-backend/pull/1172/commits/dc591e71d2efaf59bb0735973a4c1a484aea57f0)